### PR TITLE
Remove zip files from tests (part 3)

### PR DESCRIPTION
**Pull request by rkondratowicz**
Updated tests in `EnvelopeProcessorValidationTest`.

### CODE REVIEW

File: src/integrationTest/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/EnvelopeProcessorValidationTest.java [hunk 1]
```diff
@@ -36,7 +36,7 @@
 
     @Test
     public void should_throw_exception_when_zip_file_contains_fewer_pdfs() throws IOException, URISyntaxException {
```
Comment (nitinprabhu) [inline src/integrationTest/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/EnvelopeProcessorValidationTest.java]: Can this method throw just Exception ?

Comment (rkondratowicz) [reply src/integrationTest/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/EnvelopeProcessorValidationTest.java]: oops, missed this one. fixed

File: src/integrationTest/resources/zipcontents/mismatching_pdfs/metadata.json [hunk 2]
```diff
@@ -0,0 +1,54 @@
+{
```
Comment (rkondratowicz) [inline src/integrationTest/resources/zipcontents/mismatching_pdfs/metadata.json]: just extracted zip

File: src/integrationTest/resources/zipcontents/more_pdfs_than_declared/metadata_4_24-06-2018-00-00-00.json [hunk 3]
```diff
@@ -0,0 +1,23 @@
+{
```
Comment (rkondratowicz) [inline src/integrationTest/resources/zipcontents/more_pdfs_than_declared/metadata_4_24-06-2018-00-00-00.json]: same here, no changes

**Pull request merged**